services/searcher.ts

```diff
@@ -482,17 +483,71 @@ class Searcher {
       max,
       filters
     );
-    const { resultCount, took, aggregations } = searchResults;
+    const { took, aggregations } = searchResults;
     const startHydrate = Date.now();
 
     const results: SearchResult[] = await new HydrateSerializer().bulkSerialize(
       searchResults.output
     );
+
     const hydrateDuration: number = Date.now() - startHydrate;
 
+    // Do extra filtering if Honors or Campus Filters are being applied since those filter sections,
+    // which ElasticSearch doesn't actually filter out.
+
+    const validFilters = this.validateFilters(filters);
+    let honorsFilter = false;
+    let campusFilter = "";
+
+    // are we filtering by honors?
+    if (Object.keys(validFilters).includes("honors")) {
+      honorsFilter = true;
+    }
+
+    // are we filtering by campus?
+    if (Object.keys(validFilters).includes("campus")) {
+      campusFilter = "" + validFilters["campus"];
+    }
```

Comment:
```suggestion
    const campusFilter = validFilters["campus"] ?? "";
```

Same comment as above - try to remove this comment, since it doesn't add much. 

For context - the `??` is the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). If the argument on the left is `null`, it returns the argument on the right. So, if `validFilters["campus"]` doesn't exist, it defaults to `""`